test/sample/index.html

```diff
@@ -8,6 +8,7 @@
         <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
         <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css" />
         <link rel="stylesheet" href="styles/main.css">
+        <link rel="preload" href="styles/preload.css" as="style">
```

Comment:
@addyosmani in reference to https://github.com/addyosmani/oust/pull/19/files#r67332062, this is the only adjustment I made for the test. The `test.html` file was for my use case and wasn't meant to be committed.